Compass/src/Controller/GoalViewController.swift

```diff
@@ -36,14 +37,18 @@ class GoalViewController: UIViewController, UIScrollViewDelegate{
     @IBOutlet var reward: UILabel!
     @IBOutlet var author: UILabel!
     
+    @IBOutlet weak var buttonContainer: UIView!
+    
     private var rewardHeaderConstraints: [NSLayoutConstraint] = [NSLayoutConstraint]();
     private var rewardConstraints: [NSLayoutConstraint] = [NSLayoutConstraint]();
     private var authorConstraints: [NSLayoutConstraint] = [NSLayoutConstraint]();
     
     
     override func viewDidLoad(){
         //Background color of the header
-        header.layer.backgroundColor = category.getParsedColor().CGColor;
+        if let color = category?.getParsedColor().CGColor {
```

Comment:
The `?` here is not necessary, since `category` is implicitly unwrapped (the declaration has a `!`)